### PR TITLE
Feat/user discrepancies display

### PR DESCRIPTION
**Pull request by adekbadek**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

1. Enables the "Newspack Network Activity" user column on nodes (previously it was only displayed on the hub)
2. Enhances user count discrepancies debugging columns in the nodes list:

image 

(Note: this is a part of integrating the changes from the `data-integrity-improvements` branch (https://github.com/Automattic/newspack-network/pull/89))

### How to test the changes in this Pull Request:

1. On your network, create some user discrepancies by deleting some synchronized users directly in the DB (so the deletions are not synchronised), and adding some via CLI (so the additions are not synchronized) 
4. Visit the Nodes view in the hub site, appending a `_newspack_user_discrepancies` param
5. Observe the discrepancies table as visible on the screenshot above, inspect the results to verify if the data matches your specific discrepancies
6. Visit the WP Users view on both a node site and the hub site, observe the "Newspack Network Activity" column is present, with a "View All" link to the hub site's Event Log

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Review by dkoo — APPROVED** [review]
Works as described. Non-blocking question: why hide the discrepancies column behind a URL param? It seems broadly useful.

File: includes/hub/admin/class-nodes-list.php [hunk 1]
```diff
@@ -80,6 +110,38 @@ function ( $bookmark ) {
...
+			$not_on_node = array_diff( self::$hub_site_info['sync_user_emails'], $node_users_emails );
+			// Users who are not on the Node but are on the Hub.
+			$not_on_hub = array_diff( $node_users_emails, self::$hub_site_info['sync_user_emails'] );
+			if ( 'user_discrepancies' === $column ) {
+				?>
+					<span>
+						<?php
+							echo esc_html(
+								/* translators: %d - users on the Hub only, %d on the Node only */
+								sprintf( __( '%1$d on the Hub only, %2$d on the Node only', 'newspack-network' ), count( $not_on_hub ), count( $not_on_node ) )
```
Comment (dkoo) [inline includes/hub/admin/class-nodes-list.php]:
```suggestion
								sprintf( __( '%1$d on the Hub only, %2$d on the Node only', 'newspack-network' ), count( $not_on_node ), count( $not_on_hub ) )
```

**Review by dkoo — CHANGES_REQUESTED** [review]
Whoops, I think the counts are flipped. See below

**Comment by leogermani** [comment]
any reason this PR is staled?

**Comment by dkoo** [comment]
No reason—just fell off my radar.

**Review by dkoo — APPROVED** [review]

**Pull request merged**